client/src/components/categorical/value.js

```diff
@@ -173,6 +179,27 @@ class CategoryValue extends React.Component {
     });
   };
 
+  isAddCurrentSelectionDisabled(category, value) {
+    /*
+    disable "add current selection to label", if one of the following is true:
+    1. no cells are selected
+    2. all currently selected cells already have this label, on this category
+    */
+    const { crossfilter, world } = this.props;
+
+    // 1. no cells selected?
+    if (crossfilter.countSelected() == 0) {
+      return true;
+    }
+    // 2. all selected cells already have the label
+    const mask = crossfilter.allSelectedMask();
+    if (AH.allHaveLabelByMask(world.obsAnnotations, category, value, mask)) {
```

Comment:
I would request a name change for this helper...  it's accurate but I think `mask` is a concept internal to the crossfilter. `selection`?

Reply:
mask is not internal - it is a fully supported interface used all over the place, to bridge crossfilter to rendering (graph) and dataframes (subsetting, etc).

Can you suggest a better name?  I spent a lot of time trying to come up with something originally, and failed :-)

Reply:
I think this is worth making an issue for and perhaps discussing in greater length. This code, in `client/src/reducers/universe`, presents a conceptual interface which spans the concept of a `selection` and a `mask`, with little for the programmer to grasp in terms of what handoff is being made between the concepts IMO. 

![image](https://user-images.githubusercontent.com/1770265/68145491-c278a680-feea-11e9-83d2-22b1926df9e1.png)

Reply:
Since it's consistent with the rest of the codebase, I'm ok approving this as 'status quo' so long as an issue is created.